src/dodal/devices/aithre_lasershaping/cameras.py

```diff
@@ -0,0 +1,59 @@
+from enum import IntEnum
+
+from ophyd_async.core import DEFAULT_TIMEOUT, AsyncStatus, LazyMock, StandardReadable
+from ophyd_async.epics.core import epics_signal_rw
+
+from dodal.common.signal_utils import create_hardware_backed_soft_signal
+from dodal.devices.areadetector.plugins.CAM import Cam
+from dodal.devices.oav.oav_parameters import DEFAULT_OAV_WINDOW, OAVConfig
+from dodal.devices.oav.snapshots.snapshot_with_beam_centre import SnapshotWithBeamCentre
+from dodal.devices.oav.snapshots.snapshot_with_grid import SnapshotWithGrid
+from dodal.log import LOGGER
+
+# put this into a config file
+laserOAVconfig = {"microns_per_pixel_x": 6,
+                  "microns_per_pixel_y": 6,
+                  "beam_centre_x": 1000,
+                  "beam_centre_y": 1028,
+                }
+
+class OAV(StandardReadable)
```

Comment:
There's a missing colon here, you can tell at first glance in github because the `__init__` below it has no highlighting unlike other functions
```suggestion
class OAV(StandardReadable):
```